action.yml

```
@@ -46,3 +46,6 @@ inputs:
     required: false
     description: Whether to mark existing deployments of this environment as inactive
     default: 'true'
+  repository:
+    required: false
+    description: set status for a different owner/repository (optional, defaults to the same repo and owner as the action)
```

```suggestion
    description: Update a different repository's deployment status instead, using the format `${owner}/${repository}` (optional, defaults to the same repo and owner as the action).
```

It might be convenient to be able to set just the repo name as you describe in the PR:

> repo: use current owner, and set repo

But I think it's better to just be explicit in general, and always require a full `${owner}/${repository}`, and fail otherwise

---

Yeah, you're right. It over complicates the code and trying to explain that to the user 🙈 I'll take it away, it will simplify things a lot.